### ISSUE TITLE
Error: Unbound variable: CCL::COMMAND-LINE-ARGUMENTS

### ISSUE THREAD

**Issue by vindarel** [libre-man/unix-opts#12]
We found a bug here: https://github.com/next-browser/next/issues/106#issuecomment-399345489

> Error: Unbound variable: CCL::COMMAND-LINE-ARGUMENTS
> While executing: UNIX-OPTS:ARGV, in process Initial(0).
> Type :POP to abort, :R for a list of available restarts.
> Type :? for other options

This is the related code:

```
(defun handle-malformed-cli-arg (condition)
  (format t "Error parsing argument ~a: ~a.~&" (opts:option condition) condition)
  (opts:describe)
  (uiop:quit))

(defun parse-cli-args ()
  "Parse command line arguments."
  (opts:define-opts
    (:name :help
           :description "Print this help and exit."
           :short #\h
           :long "help")
    (:name :verbose
           :short #\v
           :long "verbose"
           :description "Print debugging information to stdout."))

  (handler-bind ((opts:unknown-option #'handle-malformed-cli-arg)
                 (opts:missing-arg #'handle-malformed-cli-arg)
                 (opts:arg-parser-failed #'handle-malformed-cli-arg)
                 ;; (opts:missing-required-option #'handle-malformed-cli-arg)
                 )
    (opts:get-opts)))

(defun start ()
  (multiple-value-bind (options free-args)
      (parse-cli-args)
    (when (getf options :help)
      (opts:describe :prefix "Next command line usage:")
      (uiop:quit))
    (when (getf options :verbose)
      (format t "Arguments parsed: ~a and ~a~&" options free-args))
```

**Comment by libre-man** [comment]
What version of `unix-opts`? As this is issue seems to be a replicate of #9 which was fixed on version 0.1.7.

**Comment by vindarel** [comment]
thanks, this is it. (This issue was easy to find…)
